pcsc-sharp/Interop/Unix/PCSCliteAPI.cs

```diff
@@ -25,12 +29,51 @@ internal sealed class PCSCliteAPI : ISCardAPI
         public const int MAX_ATR_SIZE = 33;
 
         private IntPtr _libHandle = IntPtr.Zero;
+        private string pcsc_lib = PCSC_LIB;
```

Comment:
Sure, I was trying to stick to the pattern. I will most gladly spell things out :)